Packages/Tracking/OpenXR/Runtime/Scripts/MetaCompatibility.cs

```diff
@@ -0,0 +1,75 @@
+using Leap;
+using System;
+using UnityEngine;
+
+namespace Leap.Tracking.OpenXR
+{
+    public static class MetaCompatibility
+    {
+        private enum ConversionType { MetaToLeap, LeapToMeta };
+
+        // These constants encode the Leap -> Meta scaling / offsets.
+        private static readonly float[] MetacarpalOutRotations = { 0.0f, 0.0f, 0.0f, 6.0f, 4.0f };
```

Comment:
Is it worth saying where these constants came from?

Reply:
Feel free to add a comment that they come from your blender interrogation I think?

Reply:
Will do.